File: Modul_4/Bahasa Python/Prak402_2310817210013MuhammadAufaFitrianda.py
angka = int(input("Masukkan Angka Anda \t: "))

for ganjil in range(1,angka + 1) :
    
    if (ganjil % 2 != 0) :
        print(ganjil, end=" ")

print()

for genap in range(angka,1,-1) :
    
    if (genap % 2 == 0) :
        print(genap, end=" ")

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_C/Prak402_2310817210013MuhammadAufaFitrianda.c
#include <stdio.h>

void main() {
    int angka;

    printf("Masukkan Angka Anda \t: ");
    scanf("%d", &angka);

    for( int ganjil = 1 ; ganjil <= angka ; ganjil++) {
        
        if( ganjil % 2 != 0) {
            printf("%d ", ganjil);
        }

    }    

    printf("\n");

    for (int genap = angka ; genap >= 1 ; genap--) {

         if (genap % 2 == 0) {
            printf("%d ", genap);
        }
        
    }
}